### DISCUSSION TITLE
Welcome to nemanjam.github.io Discussions!

### DISCUSSION THREAD

**Discussion by nemanjam** [nemanjam/nemanjam.github.io#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mazipan** [comment]
Hi, thanks for your wonderful project starter.

I have shipped the rewrite version of my personal blog using your starter couple days ago.

You've helped me to kickstart the process easier, since I am new with Astro, there are many parts I still don't understand how to do it properly, but by using your starter I already remove many obstacles in front.

As many others starter, I need to do some alignment to match with my style, here is the public repository: [mazipan/mazipan.space](https://github.com/mazipan/mazipan.space) and the live version in [mazipan.space](https://mazipan.space)

Again, I'm very grateful to found out this starter when doing my rewrite process. It's been an exciting time.

Cheers to you 👌

**Reply by nemanjam** [reply]
Thank you very much, I highly appreciate what you said. I am glad my project was useful to you. Your website looks beautiful, you took it to the next level, excellent job.
